lib/matplotlib/tests/test_animation.py

```diff
@@ -136,27 +136,14 @@ def isAvailable(cls):
     (writer, Path(output)) for writer, output in WRITER_OUTPUT]
 
 
-# Smoke test for saving animations.  In the future, we should probably
-# design more sophisticated tests which compare resulting frames a-la
-# matplotlib.testing.image_comparison
-@pytest.mark.parametrize('writer, output', WRITER_OUTPUT)
-def test_save_animation_smoketest(tmpdir, writer, output):
-    if not animation.writers.is_available(writer):
-        pytest.skip("writer '%s' not available on this system" % writer)
+@pytest.fixture()
+def simple_animation():
```

Comment:
This could maybe replace `make_animation` as well?

Reply:
This required a bit of `pytest` gymnastics that I've always wanted to figure out but never known how to do: how to pass parameters to fixtures! Turns out you can do it pretty cleanly using `indirect` nowadays. Pushed a commit to do it this way.